.eslintrc.js

```diff
@@ -88,6 +88,14 @@ module.exports = {
         'react-hooks/exhaustive-deps': 'off',
       },
     },
+    {
+      files: [
+        'src/legacy/core_plugins/vis_default_editor/public/components/controls/**/*.{ts,tsx}',
```

Comment:
All of the controls ignored as a temporary solution, I'll create an additional task to update it and exclude any possible regression and circular re-renders.